This patch file is from Qualcomm Innovation Center, Inc. and is provided under the following license:

Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
SPDX-License-Identifier: BSD-3-Clause-Clear

From d5d24b76d08e82d4aec721b8d8af55b52ef26249 Mon Sep 17 00:00:00 2001
From: Puranam V G Tejaswi <quic_pvgtejas@quicinc.com>
Date: Mon, 21 Oct 2024 16:50:41 +0530
Subject: [PATCH] freedreno: Add support for Adreno 663 GPU

Add support to enable basic functionality of Adreno 663 GPU.

Signed-off-by: quic_lkondred <quic_lkondred@quicinc.com>
Upstream-Status: Backport

diff --git a/src/freedreno/common/freedreno_devices.py b/src/freedreno/common/freedreno_devices.py
index aecc477a706..97df3743920 100644
--- a/src/freedreno/common/freedreno_devices.py
+++ b/src/freedreno/common/freedreno_devices.py
@@ -342,6 +342,7 @@ add_gpus([
 
 add_gpus([
         GPUId(660),
+        GPUId(chip_id=0xffff06060300, name="FD663"),
     ], A6xxGPUInfo(
         a6xx_gen4,
         num_sp_cores = 3,
-- 
2.34.1

